Xamarin.Forms.Platform.iOS/Renderers/TimePickerRenderer.cs

```diff
@@ -195,7 +197,32 @@ void UpdateCharacterSpacing()
 		void UpdateTime()
 		{
 			_picker.Date = new DateTime(1, 1, 1).Add(Element.Time).ToNSDate();
-			Control.Text = DateTime.Today.Add(Element.Time).ToString(Element.Format);
+			string iOSLocale = NSLocale.CurrentLocale.CountryCode;
+			var cultureInfos = CultureInfo.GetCultures(CultureTypes.AllCultures)
+							  .Where(c => c.Name.EndsWith("-" + iOSLocale)).First();
+			if (String.IsNullOrEmpty(Element.Format))
+			{
+				string timeformat = cultureInfos.DateTimeFormat.ShortTimePattern;
+				NSLocale locale = new NSLocale(cultureInfos.TwoLetterISOLanguageName);
+				Control.Text = DateTime.Today.Add(Element.Time).ToString(timeformat, cultureInfos);
+				_picker.Locale = locale;
+			}
+			else
+			{
+				Control.Text = DateTime.Today.Add(Element.Time).ToString(Element.Format, cultureInfos);
+			}
+			if (Element.Format.Contains('H'))
```

Comment:
Should this be

```C#
Element.Format?.Contains('H') == true
```

I notice a few checks around on if Format is null or empty so might be good to protect against Format being set to null?

I tested this a little bit and iOS seems to be the only platform that crashes when you set Format to null

But probably good to just sweep through those renderers see if there are any places that would be good to check for null